packages/components/psammead-media-player/src/index.stories.jsx

```diff
@@ -27,3 +27,14 @@ storiesOf('Components|Media Player', module).add('Audio Skin', () => (
     skin="audio"
   />
 ));
+
+storiesOf('Components|Media Player', module)
+  .add(ampDecorator)
+  .add('Audio Skin AMP', () => (
+    <AmpMediaPlayer
+      isAmp
+      src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp"
+      placeholderSrc="https://ichef.bbci.co.uk/news/640/cpsdevpb/4eb7/test/ba7482d0-cca8-11e8-b0bf-f33155223fc4.jpg"
```

Comment:
```suggestion
      placeholderSrc="https://news.files.bbci.co.uk/include/articles/public/images/audio-player-placeholder.png"
```

The audio placeholder image is now available 😄

Reply:
Done 👍